src/panels/config/info/system-health-card.ts

```diff
@@ -128,30 +128,28 @@ class SystemHealthCard extends LitElement {
   }
 
   private _copyInfo(): void {
-    // We have to copy title text and content separately, because
-    // copying the whole <ha-card> would also copy the tooltip text.
-    const selection = window.getSelection()!;
-    selection.removeAllRanges();
-
-    let copyElement = this.shadowRoot?.querySelector(
-      ".card-header-text"
-    ) as HTMLElement;
-
-    let range = document.createRange();
-    range.selectNodeContents(copyElement);
-    selection.addRange(range);
-
-    copyElement = this.shadowRoot?.querySelector(
+    const copyElement = this.shadowRoot?.querySelector(
       ".card-content"
     ) as HTMLElement;
 
-    range = document.createRange();
+    // Add temporary heading (fixed in EN since usually executed to provide support data)
+    copyElement.insertAdjacentHTML(
+      "afterbegin",
+      '<h3 id="temporary">System Health</h3>'
+    );
```

Comment:
```suggestion
	const tempTitle = document.createElement("h3");
	tempTitle.innerText = "System Health";
	copyElement.prepend(tempTitle);
	copyElement.removeChild(tempTitle);
```

Reply:
Or use `insertBefore` as I'm not sure `prepend` is supported in all browsers

Reply:
Yes `prepend` is still experimental, so I went with `insertBefore`.